HabitRPG/HabitRPG.xcdatamodeld/HabitRPG 47.xcdatamodel/contents

```diff
@@ -36,12 +36,17 @@
         <relationship name="user" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="User" inverseName="challenges" inverseEntity="User" syncable="YES"/>
     </entity>
     <entity name="ChallengeTask" representedClassName="ChallengeTask" syncable="YES">
+        <attribute name="attribute" optional="YES" attributeType="String" syncable="YES"/>
```

Comment:
Adding new fields to this model will cause issues when users update the app. There should be a new model version with the changes, so that core data can migrate

Reply:
Great! Will do.